### PR TITLE
Fix ATM fraud example

### LINKED ISSUE #89: Update the ATM example to use an up-to-date configuration.
The example ATM pipeline's `config.yaml` is broken due to the use of at least 1 key that is deprecated. 
```
kafka_connect_resetter_config:
    namespace: "kpops"
```

Currently the example doesn't work as is, update it so that it does.

### PR DESCRIPTION
**Pull request by disrupted**
Fixes #89 
due to recent changes, this was no longer working

### CODE REVIEW

**Review by philipp94831 — COMMENTED** [review]
Should we add a workflow that at least runs generate on this pipeline in order to test it?

**Comment by raminqaf** [comment]
fixes https://github.com/bakdata/kpops/issues/89?

**Review by philipp94831 — APPROVED** [review]

File: tests/pipeline/test_pipeline.py [hunk 1]
```diff
@@ -287,3 +287,29 @@ def test_default_config(self, tmpdir, snapshot):
         assert error_topic == "resources-custom-config-app2-error"
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
+
+    def test_atm_fraud_example(self, tmpdir, snapshot):
```
Comment (raminqaf) [inline tests/pipeline/test_pipeline.py:291]: Should we move this test to its own file? like `test_atm_fraud_pipelie.py` So we have a new snapshot from it?
I have this feeling that the massive snapshot is hard to debug and understand sometimes.

Comment (disrupted) [reply tests/pipeline/test_pipeline.py:291]: good idea, can I do it in the followup #96? I have already refactored the test fixtures there

**Pull request merged**